msal4j-sdk/src/integrationtest/java/com.microsoft.aad.msal4j/AcquireTokenInteractiveIT.java

```diff
@@ -208,42 +208,6 @@ private IAuthenticationResult acquireTokenSilently(IPublicClientApplication pca,
                 .get();
     }
 
-    public void acquireTokensInHomeAndGuestClouds(String homeCloud) throws MalformedURLException {
```

Comment:
Surprised to just see a whole test getting removed - what's up?

Reply:
It wasn't a whole test, it was a helper for some tests (like the similarly named "acquireTokenInteractive" and "acquireTokenInteractive_instanceAware")

However, it it seems like other changes and refactoring over the years got rid of all usages of it and it must've been missed by whatever got rid of the last reference.